docs/core/compatibility/2.2-3.1.md

```diff
@@ -279,18 +364,57 @@ If you're migrating from version 2.2 to version 3.1 of .NET Core, ASP.NET Core,
 
 [!INCLUDE[Better argument validation in the Pkcs8PrivateKeyInfo constructor](~/includes/core-changes/cryptography/3.0/better-argument-validation-in-pkcs8privatekeyinfo-ctor.md)]
 
+***
+
 ## Entity Framework Core
 
 [Entity Framework Core breaking changes](/ef/core/what-is-new/ef-core-3.0/breaking-changes)
 
+***
+
 ## Globalization
 
+- ["C" locale maps to the invariant locale](#c-locale-maps-to-the-invariant-locale)
+
+***
+
 [!INCLUDE["C" locale maps to the invariant locale](~/includes/core-changes/globalization/3.0/c-locale-maps-to-invariant-locale.md)]
 
+***
```

Comment:
it's interesting that this line doesn't have spacing between the text and the line even though there is one here. not sure what's different with some of these includes that some have the extra spacing and some don't.